job/runner.go

```diff
@@ -53,6 +53,18 @@ import (
 	log "github.com/sirupsen/logrus"
 )
 
+type CleanupType int
+
+const (
+	All CleanupType = iota
```

Comment:
__[Golint]__ exported const All should have comment (or a comment on this block) or be unexported ([link](https://sider.review/gh/repos/169710725/pulls/25#issue-4259315))

<sub>posted by Sider</sub>